samples/blog/BlogArticles.scala

```diff
@@ -13,6 +13,11 @@ object BlogArticlesApi {
     case object DRAFT extends BlogArticlesApi.ArticleStatus
     case object PUBLISHED extends BlogArticlesApi.ArticleStatus
   }
-  case class Pagination(first: Int, count: Int, reverse: Option[Boolean])
+  case class Pagination(first: Int, count: Int, order: Option[PaginationOrder])
```

Comment:
`Pagination` and `PaginationOrder` are generated multiple times, can we reduce the duplication here?